src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/AbstractDocumentDetails.java

```diff
@@ -60,4 +63,9 @@ public LocalDateTime getDateTimeFormatted() {
             return null;
         }
     }
+
+    @JsonIgnore
+    public String getDocumentDetailsSubType() {
```

Comment:
Instead of this, we could use something like `doc.getValue().isAssignableFrom(SscsDocument.class)` in the client code

Reply:
tried in the client and works 👍 . Updated